internal/configs/configurator.go

```diff
@@ -186,7 +187,7 @@ func (cnf *Configurator) AddOrUpdateIngress(ingEx *IngressEx) error {
 		return fmt.Errorf("Error adding or updating ingress %v/%v: %v", ingEx.Ingress.Namespace, ingEx.Ingress.Name, err)
 	}
 
-	if err := cnf.nginxManager.Reload(); err != nil {
+	if err := cnf.nginxManager.Reload(collectors.ReloadForOtherUpdate); err != nil {
```

Comment:
it is a bit strange that we call nginxManager from `nginx` package here and at the same time we use a constant from a different package. Could you move those  two constants to `nginx` package? (that was the original suggestion)